validphys2/src/validphys/process_options.py

```diff
@@ -10,7 +10,7 @@
 import numpy as np
 from validobj.custom import Parser
 
-TMASS = 173.3
+TMASS = 172.5
```

Comment:
Please don't change this because this was the value that was there in the old version so until _all_ processes are migrated this should not be touched.
If you need the value of the mass of the top add it to the kinematic information of the relevant dataset.